vnext/proposals/active/keyboard-reconcile-desktop.md

```diff
@@ -0,0 +1,148 @@
+
+| **TYPE** | **REQUIRED** | **PLATFORM** | **DEFAULT** |
+| --- | --- | --- | --- |
+| boolean | No | Windows, macOS | false |
+
+React Native core introduced `focusable` that does the same thing as acceptsKeyboardFocus in v0.62. In order to align and make this complete, the work here includes the following:
+- **COMPLETED** : Windows has added `focusable` and marked `acceptsKeyboardFocus` as deprecated in 0.62. 
+- macOS needs to do add `focusable` and mark `acceptsKeyboardFocus` as deprecated. Tracked by - [Issue#498](https://github.com/microsoft/react-native-macos/issues/498)
+- **COMPLETED** : Add focusable to Pressable ([windows Issue#5512](https://github.com/microsoft/react-native-windows/issues/5512), [macOS Issue#500](https://github.com/microsoft/react-native-macos/issues/500)). 
+- `focusable` is already supported in Android and exists in core. There is no hard upstreaming requirement, however – it would be good to implement this for iOS/iPadOS for completion.
+
+## focus, blur, onFocus, onBlur on View
+
+React Native core supports `.focus()`, `.blur()` methods and `onFocus()`, `onBlur()` callbacks on TextInput component. Windows and macOS implementations add the same methods and callbacks on View in addition to supporting them on TextInput. The work remaining for these include:
+- **COMPLETED** : The event bubbling behavior with .focus() and .blur() on View in macOS has some differences from Windows and w3c standards. The behavior needs to be evaluated and reconciled between the 2 platforms.
+- The microsoft/react-native fork has implementations for all the above methods and callbacks for iOS and Android as well. Once the event bubbling behavior above has been reconciled, these can be upstreamed to core. Tracked by [Issue#243](https://github.com/microsoft/react-native-macos/issues/243)
+- TextInput also includes an `isFocused()` method that returns true/false based on whether this TextInput currently has focus. It will be useful to add this to View to and upstream as a separate PR. Tee this up with the previous one. Tracked by - [windows Issue#5511](https://github.com/microsoft/react-native-windows/issues/5511), [macOS Issue#499](https://github.com/microsoft/react-native-macos/issues/499)
+- Add `.focus()`, `.blur()`, `onFocus()`, `onBlur()` and `isFocused()` to Pressable. Tracked by [windows Issue#5589](https://github.com/microsoft/react-native-windows/issues/5589), [macOS Issue#518](https://github.com/microsoft/react-native-macos/issues/518)
+
+## Default keyboarding behaviors on JS components
+
+Both Windows and macOS implementations have added appropriate keyboarding defaults to JS components like Button, Touchables. The implementation includes firing onPress() callbacks on Button and Touchable components when they are in focus and Enter/Space key is pressed. The work remaining for these include:
```

Comment:
@harinikmsft @rectified95 Open question about this that comes from an issue of our own implementation right now. `PressEvent` exposes bits like pointer coordinates that `KeyEvent`s won't have. How do we reconcile these?